src/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+# List any Python dependencies for the image here
+[packages]
+# This should match the version of the image
+example = {file = "https://github.com/cisagov/skeleton-python-library/archive/v0.0.1.tar.gz"}
+
+# This version should match the version of Python in the image
+[requires]
+python_full_version = "3.12.0"
```

Comment:
Is there a good way to test that these versions match what they are supposed to?

Reply:
That is actually being done here: https://github.com/cisagov/skeleton-docker/blob/8e03ad95147adeb44bd2773b146eb6f752f53b6c/Dockerfile#L53
If the versions do not match it will fail and the image build will fail in turn.

Reply:
Excellent, I overlooked that - thanks!